raiden-dapp/src/App.vue

```diff
@@ -43,20 +43,27 @@ import ReceivingDisabledDialog from '@/components/dialogs/ReceivingDisabledDialo
     AppHeader,
     OfflineSnackbar,
     UpdateSnackbar,
-    ReceivingDisabledDialog,
+    NotificationArea,
   },
 })
 export default class App extends Mixins(NavigationMixin) {
   canReceive!: boolean;
-  showReceivingDisabled = false;
 
   destroyed() {
     this.$raiden.disconnect();
   }
 
   @Watch('canReceive')
   onCanReceiveChanged(canReceive: boolean | undefined) {
-    this.showReceivingDisabled = canReceive === false;
+    if (canReceive !== false) {
+      return;
+    }
+    this.$store.dispatch('notifications/notify', {
```

Comment:
Since  this is just a dispatching, maybe we should take away config and getter from store, and instead dispatch it directly from RaidenService? No need for a watcher in a random place to dispatch an action if we already can get it directly from the observable's emission.